lua/nvim-lsp-installer/servers/fsautocomplete/init.lua

```diff
@@ -0,0 +1,25 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local std = require "nvim-lsp-installer.installers.std"
+local shell = require "nvim-lsp-installer.installers.shell"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "fsharp" },
+        homepage = "https://fsharp.org",
+        installer = {
+            std.ensure_executables {
+                { "dotnet", "dotnet was not found in path. fsautocomplete is installed as global dotnet tool." },
+            },
+            shell.polyshell[[dotnet tool update --tool-path ./ fsautocomplete]],
```

Comment:
Super minor, but this makes it not rely on Unix path separators (so that this can run on Windows too)

```suggestion
            shell.polyshell[[dotnet tool update --tool-path . fsautocomplete]],
```